lib/std/fmt.zig

```diff
@@ -876,8 +876,11 @@ fn formatSizeImpl(comptime radix: comptime_int) type {
             writer: anytype,
         ) !void {
             _ = fmt;
+            // The worst case in terms of space needed is... TODO.
+            var buf: [1000]u8 = undefined;
```

Comment:
The worst case for `formatFloatDecimal` seems to be 32 bytes + 3 bytes for the suffix.